server/test/red-flag.spec.js

```diff
@@ -53,6 +61,7 @@ describe('Ride-My-Way', () => {
       const redFlag = db[4];
       const response = await server.patch(`/api/v1/red-flags/${redFlag.id}/comment`);
       expect(response.body.status).to.equal(200);
+      expect(response.body.data[0].message).to.eql('Updated red-flag record\'s comment');
```

Comment:
You should also check that the updated data is returned with the updated values.